src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
@parjong I think that it could be done by pushing placeholders for the additional one or two arguments right in the jitted code and replacing them with the real values in the stub. So the call site e.g. for the case with two additional args would look like this:
```asm
push 0 // placeholder for the arg1
push 0 // placeholder for the arg2
call stub
```
and the stub would then look something like:
```asm
mov dword ptr [rsp+8], arg1
mov dword ptr [rsp+4], arg2
jmp target
```